src/main/app/forms/validation/validators/fractions.ts

```diff
@@ -24,6 +24,11 @@ export class FractionsConstraint implements ValidatorConstraintInterface {
     if (value == null) {
       return true
     }
+
+    if (typeof value !== 'number') {
+      return true
```

Comment:
As this Validator is generic to be used elsewhere, does not look right to say non-numbers are valid fractions.

Reply:
not this validator's concern.
IsNumber for that

Reply:
As we are making the Fraction validator to be used with @isNumber to work properly. Doesn't feel right. But we can go with this, is not a blocker.